functions/src/models/iot-resource-object.ts

```diff
@@ -0,0 +1,28 @@
+import {
+  AuthClaims
+} from "../firestore-types";
+import {BaseResourceObject} from "./base-resource-object";
+import {
+  Credentials, Config, IotResource
+} from "../resource-types";
+
+export class IotResourceObject extends BaseResourceObject {
+  // TODO: add iot specific resource members
```

Comment:
Was this a TODO for this PR or for later?

Reply:
Same question with the other TODOs

Reply:
Thanks for asking. These are for later. They were actually there before the refactoring. We never actually used this IotResource in the DataFlow project. But I'm about to add a AthenaWorkgroup Resource which hopefully we will use, so the IotResource served as a good place holder to help with the separation.